content/en/docs/setup/release/notes.md

```diff
@@ -1,4 +1,15 @@
+---
+title: Release Notes
+content_template: templates/concept
+---
+{{% capture overview %}}
+
+[Documentation](https://docs.k8s.io) & [Examples](https://releases.k8s.io/release-1.12/examples)
+
```

Comment:
Why add this line?

Reply:
In this commit #10258 ,this line still exists.
In next commit #10292 ,it It disappeared.